values.yaml

```diff
@@ -88,8 +88,8 @@ global:
   # Deploy auth sidecar to use openshift native features
   authSidecar:
     enabled: false
-    repository: nginxinc/nginx-unprivileged
-    tag: stable
+    repository: quay.io/astronomer/ap-auth-sidecar
```

Comment:
Are we ready to make this change in 0.25 release? What is the risk involved?

Reply:
Yes since the ap-auth-sidecar  is built on top of nginxinc/nginx-unprivileged and we are safe to make this change